src/Console/Command/ObfuscateData.php

```diff
@@ -0,0 +1,305 @@
+<?php namespace Ushahidi\Console\Command;
+
+/**
+ * Ushahidi Obfuscate Data console command
+ *  - This overwrites specific data within the current database
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Console
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Illuminate\Console\Command;
+use Ushahidi\Core\Entity\PostRepository;
+use Ushahidi\Core\Entity\ContactRepository;
+use Ushahidi\Core\Entity\ConfigRepository;
+use Ushahidi\Core\Entity\UserRepository;
+use Ushahidi\App\Multisite;
+use Ohanzee\DB;
+use Faker;
+use Database;
+
+class ObfuscateData extends Command
+{
+    private $postRepository;
+    private $contactRepository;
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'db:obfuscatedata';
+
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'db:obfuscatedata';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Obfuscates selected PII and removes various config keys from database';
+
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    public function handle()
+    {
+        //set up repos
+        $this->contactRepository = service('repository.contact');
+        $this->postRepository = service('repository.post');
+        $this->configRepository = service('repository.config');
+        $this->userRepository = service('repository.user');
+
+        $this->db = service('kohana.db');
+
+        if ($this->isThisAMultisiteInstall()) {
```

Comment:
Is there something we can call usefully on the Multisite service to answer this? and check if we have `HOST` define?

If not, maybe move this code there instead.